src/compiler/diagnosticMessages.json

```diff
@@ -5793,10 +5617,6 @@
         "category": "Message",
         "code": 95001
     },
-    "Convert function '{0}' to class": {

```

Comment:
I'm almost certain this message and the ones that follow are used in codefixes. I wonder why our fourslash tests don't fail with them gone.